.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    # trying and staging branches are for BORS config
+    branches:
+      - trying
+      - staging
+      - main
+
+jobs:
+  linter:
+    name: linter on node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '12', '14' ]
```

Comment:
No need to test the linter on different node versions -> this is only useful for the tests.
The linter is here to check your code is clear and clean, no need to check the linter command works for different node version